app/authentication/auth.py

```diff
@@ -21,8 +21,20 @@
 JWT_AUTH_TYPE = "jwt"
 API_KEY_V1_AUTH_TYPE = "api_key_v1"
 AUTH_TYPES = [
-    ("Bearer", JWT_AUTH_TYPE),
-    ("ApiKey-v1", API_KEY_V1_AUTH_TYPE),
+    (
+        "Bearer",
+        JWT_AUTH_TYPE,
+        "JWT token that the client generates and passes in, "
+        "see https://docs.notifications.service.gov.uk/rest-api.html#authorisation-header "
```

Comment:
Is there anything similar on https://documentation.notification.canada.ca/ that we can link to instead of gov.uk?

Reply:
Not related to JWT tokens unfortunately, perhaps there is another ask here related to adding that to our documentation ?